fpga_builder/utils.tcl

```diff
@@ -221,7 +222,10 @@ proc build {proj_name top_name proj_dir} {
   set bitstream ${proj_dir}/${proj_name}.runs/impl_1/${top_name}.bit
   
   # ------------------------------------------------------------------------------------- #
-
+  set report_origin ${proj_dir}/${reports}
```

Comment:
Does `reports` really need passed in?  If it is constant across vivado builds we can keep it in the tcl to avoid needing a new param.